### DISCUSSION TITLE
Deleting a collection does not delete metadata

### DISCUSSION THREAD

**Discussion by zemul** [seaweedfs/seaweedfs#2428] [Q&A]
We use collection to isolate different user Spaces, use Filer to upload files, and store metadata in mysql

However, we found that Mysql row records were not deleted when the collection was deleted

**Comment by chrislusf** [comment]
If using normal collections, deleting a volume would not delete metadata.

For your case, you can use S3 API to use buckets for the isolation. Each user's data and metadata would be automatically grouped into a bucket.

Or actually you can directly use Filer API, as long as the folders for each user's folder is under "/buckets". You do not need to specify collection name.
